draft-foudil-securitytxt.md

```diff
@@ -226,7 +229,7 @@ Signature: https://example.com/.well-known/security.txt.sig
 
 ## Web-based services
 
-Web-based services SHOULD place the security.txt file under the /.well-known/ path; e.g. https://example.com/.well-known/security.txt.
+Web-based services SHOULD place the security.txt file under the /.well-known/ path; e.g. https://example.com/.well-known/security.txt. A security.txt file located under the top-level path SHOULD either redirect to the security.txt file under the /.well-known/ path or be used as a fall back.
```

Comment:
Do we want to explain what "fall back" means? Same for line 69 above.

Perhaps "A security.txt file located under the top-level path SHOULD redirect to the security.txt file under the /.well-known/. **If it is not possible to place the security.txt file in the /.well-known/ path or setup a redirect, web-based services MAY place the file in the top-level path as a fall back option***"